### DISCUSSION TITLE
Mixed navigation

### DISCUSSION THREAD

**Discussion by philthebass** [nolimits4web/swiper#5695] [Q&A]
Hi all,

I'm new to Swiper and using it on a project at the moment.

I'm using it with Wordpress and Oxygen Builder and so far so good.

Is it possible to mix navigation components? I'd like to have both the bullet nav and a page slide counter as well.

I have this for displaying the bullets

`jQuery(document).ready( function(){
var swiper = new Swiper(".mySwiper", {
        direction: "vertical",
        slidesPerView: 1,
        spaceBetween: 0,
		speed: 1500,
        mousewheel: true,
		pagination: {
          el: ".swiper-pagination",
          clickable: true,
        },
      });
	
	});
	`
Is it possible to add the slide counter like this demo so I have both the bullets and counter https://codesandbox.io/s/jx6vbg?file=/index.html:2384-2401

Thanks

**Comment by Arhell** [comment]
https://swiperjs.com/demos#pagination-fraction
API
https://swiperjs.com/swiper-api#pagination

**Reply by philthebass** [reply]
Thanks Ihor, but can I mix them together in the same slider so I have both the bullets _and_ the fraction elements?

**Reply by philthebass** [reply]
I tried mixing them up like this but it wouldn't work

`jQuery(document).ready( function(){
var swiper = new Swiper(".mySwiper", {
        direction: "vertical",
        slidesPerView: 1,
        spaceBetween: 0,
		speed: 1500,
        mousewheel: true,
		pagination: {
          el: ".swiper-pagination",
			type: "bullets",
          clickable: true,
        },
		pagination: {
          el: ".swiper-counter",
          type: "fraction",
        },
		
      });
	
	});`

**Reply by Arhell** [reply]
What version are you using?
Because version 8 doesn't need jQuery

**Reply by philthebass** [reply]
I'm using 8.0.3 from the CDN - swiper@8.0.3/swiper-bundle.min.js
I'm using it with Wordpress and Oxygen Builder and adding a jQuery code block was the only way I could get it to work

**Reply by Arhell** [reply]
I pasted your config here
https://codepen.io/swmj/pen/RwPxQzR
Everything is working.

But not testing on Wordpress and Oxygen Builder

**Reply by philthebass** [reply]
Thaks Ihor, but that's not quite how I meant.

My slider is set up vertically with the bullet navigation which all works fine.

What I'm trying to figure out is how I can also also include a slide counter. I have a place holder div set up ready, but I can't figure out how to have both in my slider.

![image](https://user-images.githubusercontent.com/13836637/166950142-01b5c426-0e51-48f1-b380-b4af3589a14c.png)

**Reply by Arhell** [reply]
Unfortunately, I did not find such an option.
You can try to do it yourself
https://swiperjs.com/demos#pagination-custom

https://github.com/nolimits4web/swiper/issues/3054#issuecomment-748854609

**Reply by philthebass** [reply]
Thanks. I've been trying to combine both by using a custom class for the slide counter, but haven't been able to figure it out yet. 

I'll keep trying.

Thanks for your help

**Reply by Arhell** [reply]
You are welcome.
